src/libraries/sendtohelix-wasm.targets

```diff
@@ -231,6 +231,10 @@
         <PayloadArchive>%(Identity)</PayloadArchive>
         <Command>$(HelixCommand)</Command>
         <Timeout>$(_workItemTimeout)</Timeout>
+        <Timeout Condition="'%(FileName)' == 'System.Text.Json.Tests'">01:20:00</Timeout>
```

Comment:
Could you just add a comment pointing that the section is duplicated for `_WasmWorkItem` and `_DefaultWorkItems` to both places?

Reply:
done